data/incipias/first contact.txt

```diff
@@ -0,0 +1,287 @@
+		has "Incipias: First Contact IV: done"
+	on enter Dom'us
+		conversation
+			`This time as you enter this system, you are prepared. You stop your ship and pull out the translation device the Quarg gave you. Then you stand there and think, what would be the best first contact message?`
+			choice
+				"Hello Incipiae, I am <first> <last> of the human race a space ship captain."
+				"Greetings strangers, I come to spread peace."
+				"Hello beings of another kind, I want to communicate with you."
+			`    At first nothing happens. Then your scanners register a huge raise in communication between the alien ships, their planet and their spacestation. At first it seems to be very chaotic but then slowly the spacestation dominates the communication. Then there is nothing.`
+			choice
+				"[wait for some time]"
+			`    After some time the space station opens a communication channel to you. A creature like you never seen one before appears on the screen. Formed like a balloon, with some weird shaped extremeties and four eyes? at the bottom.It is floating in front of a blank wall. You remember what the Quarg told you and hold the tablet in the direction of the screen. Something on the surface of the creature begins flickering and words start appearing on the tablet screen. "Welcome, [stranger, traveller, alien] [I, we] are [happy, surprised, scared] by your visit." It seems like this is the difference the Quarg talked about, the translation device is trying to combine words to give you the actual meaning. More words appear, "[I, we] see another [intelligence, feeling one] the first time, please [explain, show, tell us] the reason for your presence."`
+			choice
+				"I am exploring the limits of space."
+					goto limits
+				"I am searching for new species and technologies"
+			`    The Incipias listens to you and then it looks like they are looking to someone on its right you do not see. After a short moment it turns towards you again. It starts flickering "We do not think we have [better, more advanced] technologies than [you, your kind]. But [I, we] look forward to a [benifical, peaceful] contact with [you, your kind]. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this adventure] further.`
+				goto c1
+			label limits
+			`    The Incipias listens to you and then it looks like it is looking to someone on their right you do not see. After a short moment it turns towards you again. It starts flickering "[I, we] are happy to [greet, meet, see] you traveller. [I, we] would like to invite you to our [space station, home, safe place], to [communicate, continue this meeting] further.`
+			label c1
+			`    Some basic information about the spacestation is send to your display and the communication channel gets closed.`
+			action
+				"reputiation: Hicemus" = 1
+				"reputation: Incipias" = 1
+
+mission "Incipias: First Contact VI"
+	landing
+	invisible
+	to complete
+		never
+	source "Pon'tes"
```

Comment:
no to offer?

Reply:
You can only land after you got a better rep

Reply:
Okay but its number 6?
Also did you make sure one cant farm rep by refueling?

Reply:
I am not sure what you mean?

Reply:
> Okay but its number 6? Also did you make sure one cant farm rep by refueling?

not really, they will never run out of fuel, becuase they do not use fuel